common/c/targets/esp8266/ld/eagle.app.v6.ld

```diff
@@ -160,11 +160,14 @@ SECTIONS
   {
     _irom0_text_start = ABSOLUTE(.);
     *(.irom0.literal .irom.literal .irom.text.literal .irom0.text .irom.text)
-    *libkaac_s.a:(.literal .text .literal.* .text.* .stub .gnu.warning .gnu.linkonce.literal.* .gnu.linkonce.t.*.literal .gnu.linkonce.t.*)
+    *libkaac.a:(.literal .text .literal.* .text.* .stub .gnu.warning .gnu.linkonce.literal.* .gnu.linkonce.t.*.literal .gnu.linkonce.t.*)
```

Comment:
Well, but `libextension_*` can be used to reduce the number of lines. I also don't believe `libextension_*` is needed as `libkaac` must include them.